x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -261,10 +261,7 @@ export class ActionExecutor {
             source,
           });
         } catch (err) {
-          if (
-            err.reason === ActionExecutionErrorReason.Validation ||
```

Comment:
Checking for validation error here is redundant. Validation is not in this try-catch anymore.